rts/lra/test/tck/pom.xml

```diff
@@ -72,36 +59,4 @@
         </dependency>
     </dependencies>
 
-    <profiles>
-        <profile>
-            <!-- when arq profile is activated we expect to start coordinator -->
-            <id>arq</id>
-            <properties>
-                <lra.coordinator.exec.plugin.phase>pre-integration-test</lra.coordinator.exec.plugin.phase>
-            </properties>
-        </profile>
-        <profile>
-            <id>debug.lra.tck.suite</id>
-            <activation>
-                <property>
-                    <name>debug.tck</name>
```

Comment:
I also used this a lot in the past. I liked that I can easy just add `-Ddebug.tck` or `-Ddebug.coordinator`.

Reply:
Thanks